### PR TITLE
show status for static hosting build

### PR DESCRIPTION
**Pull request by jrhizor**

### CODE REVIEW

**Comment by jrhizor** [comment]
You can look at the comment history for an example of this working on failure.

File: action/src/plugins/pluginRunner.ts [hunk 1]
```diff
@@ -6,12 +6,19 @@ import { runStaticHostingPlugin } from './staticHosting';
 
 export const runPlugins = async (
   stoatConfig: StoatConfigSchema,
+  stepsSucceeded: boolean,
   githubActionRun: GithubActionRun,
```
Comment (tuliren) [inline action/src/plugins/pluginRunner.ts:10]: I think that adding an extra parameter in the signature is unnecessary. Ideally we don't want to add an extra parameter each time there is a new feature.

The `githubActionRun` has a `ghJob` field that is the workflow job object. The step success information is included in that object. My first proposal is to move the calculation of the `stepsSucceeded` from the `app.ts` to the static hosting plugin function based on the `ghJob` field.

If the first proposal is overkill, or you prefer to log the job status in the main `app.ts`, my second proposal is that the `stepSucceeded` variable can be added as a new field under `GithubActionRun`. This approach makes sense in that the success status is a metadata for the action run. Also this avoid adding an extra parameter in the signature.

**Review by tuliren — COMMENTED** [review]
Left one comment.

File: action/src/plugins/pluginRunner.ts (hunk 1 above)
Comment (jrhizor) [reply action/src/plugins/pluginRunner.ts:10]: I'll do the latter.

Comment (jrhizor) [reply action/src/plugins/pluginRunner.ts:10]: We will probably want to use it for non-static hosting plugins in the future.

**Review by tuliren — APPROVED** [review]

**Pull request merged**